app/app.go

```diff
@@ -226,11 +233,11 @@ func New(
 		interfaceRegistry: interfaceRegistry,
 		invCheckPeriod:    invCheckPeriod,
 		keys:              keys,
-		tkeys:             tkeys,
+		tkeys:             transientKeys,
 		memKeys:           memKeys,
 	}
 
-	app.ParamsKeeper = initParamsKeeper(appCodec, legacyAmino, keys[paramstypes.StoreKey], tkeys[paramstypes.TStoreKey])
+	app.ParamsKeeper = initParamsKeeper(appCodec, legacyAmino, keys[paramstypes.StoreKey], transientKeys[paramstypes.TStoreKey])
```

Comment:
we could consider disabling this linter... (`lll`)

Reply:
I want to encourage (enforce?) no more than 120 characters preferably.